_content/monographs/logic-epistemology-theravada_hegoda-khemananda.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Logic and Epistemology in Theravada"
+authors:
+  - "Hegoda Khemananda"
+external_url: "https://archive.org/download/logic-and-epistemology-in-theravada/Logic%20and%20Epistemology%20in%20Theravada_text.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1-7xj6Sl-G3Gw1OaQupet_yAaeX1_K1SI/view?usp=drivesdk"
+translator: "Asangha Tilakaratne"
+course: abhidhamma
+tags:
+  - logic
+  - philosophy
+  - anatta
```

Comment:
Curious to hear why you included anatta?

Reply:
As I scanned the book, I felt many passages made the point that no-self is the thing to be known or experienced. Would you like to remove it?

Reply:
🤔 I feel that's a bit redundant with "epistemology" as typically they're grappling with knowing "ultimate" (as opposed to conventional) reality.